docs/content/contribution-guidelines/operations/docs-styleguide.md

```diff
@@ -13,11 +13,11 @@ This **Style Guide** is intended to complement the Design Guide to govern the _p
 
 As a starting point, here are some very basic items to consider when working on documentation for KubeStellar and its components:
 
--  [Use clear and concise prose](#use-clear-and-concise-prose)
--  [Avoid use of emojis (especially in headings)](#avoid-use-of-emojis-in-prose)
--  [Always include alt-text for images](#always-include-alt-text-for-images)
--  [Use care if using generative AI](#use-care-if-using-generative-ai) for writing assistance. Review text carefully to make sure it is both coherent and accurate.
--  [Always check spelling and grammar](#always-do-a-spelling-and-grammar-check) before committing docs changes
+- [Use clear and concise prose](#use-clear-and-concise-prose)
+- [Avoid use of emojis (especially in headings)](#avoid-use-of-emojis-in-prose)
+- [Think Accessibility](#compose-and-format-for-accessibility)

```

Comment:
It would be even better if the words that the reader sees here match the words in the section title.